app/assets/stylesheets/categories.scss

```diff
@@ -1,13 +1,25 @@
-.coffee-hero-unit {
-  background-image: url('https://images.unsplash.com/7/color-correct-coffee.JPG?crop=entropy&dpr=2&fit=crop&fm=jpg&h=675&ixjsv=2.1.0&ixlib=rb-0.3.5&q=50&w=1075');
+.sports-hero-unit {
+  background-image: url("https://upload.wikimedia.org/wikipedia/commons/7/7a/Pollock_to_Hussey.jpg");
 }
 
-.tools-hero-unit {
-  background-image: url('https://images.unsplash.com/photo-1426174840074-541ae41efdb9?ixlib=rb-0.3.5&q=80&fm=jpg&crop=entropy&w=1080&fit=max&s=321f7c741f817419062b848dd3a0c8db');
+.comedy-hero-unit {
+  background-image: url("https://i.ytimg.com/vi/Ce9f5Xj35zY/maxresdefault.jpg");
 }
 
 .festivals-hero-unit {
-  background-image: url('http://www.nothingliketheholidays.com/wp-content/uploads/2015/04/53da64d2740fb.jpg');
+  background-image: url("http://www.nothingliketheholidays.com/wp-content/uploads/2015/04/53da64d2740fb.jpg");
+}
+
+.popups-hero-unit {
+  background-image: url("http://originaleating.com/content/uploads/2013/11/Paleo-Brussels-Sprouts-with-Cranberries-by-Kate-Taylor.jpg");
```

Comment:
Going to ignore "protocol or domain" complaints. These are all URLs for pictures in hero units.